source/includes/_filters.md

```diff
@@ -3,7 +3,7 @@
 Tip — use these endpoints to help you filter resources!
 </aside>
 
-## Get All Users
+## Get All Users (v1)
```

Comment:
This and similar fixes are needed because we already have the heading with exactly the same name in v2, which causes a issues with navigation(links)